homeassistant/components/myq/cover.py

```diff
@@ -115,7 +115,9 @@ async def async_close_cover(self, **kwargs):
         # Write closing state to HASS
         self.async_write_ha_state()
 
-        if not await wait_task:
+        result = wait_task if isinstance(wait_task, bool) else await wait_task
```

Comment:
I had to read that a few times as I was a bit surprised that this returns an awaitable or a boolean.

https://github.com/arraylabs/pymyq/blob/master/pymyq/device.py#L164